tests/integration_test/services_test.py

```
@@ -10024,6 +10019,55 @@ def test_delete_collection_data():
+
+    actual = paginated_response.json()
+    assert paginated_response.status_code == 200
+    assert actual["error_code"] == 0
+    assert actual["success"]
+    assert isinstance(actual["data"], list)
+
+    # 4. Compare paginated response with expected slice
+    assert actual["data"] == expected_slice
+
+    # 5. Verify record order matches
+    for a, b in zip(actual["data"], expected_slice):
```

Ruff (0.14.4)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)